content/applications/hr/payroll/contracts.rst

```diff
@@ -0,0 +1,250 @@
+   :alt: Edit the details for the contract.
+
+- :guilabel:`Tags`: Select any tags associated with the contract.
+- :guilabel:`Signed Document Workspace`: This is where the signatures are stored. Choose a
+  pre-configured workspace or create a new one.
+- :guilabel:`Signed Document Tags`: Select or create any tags associated only with the signed
+  contract as opposed to the original unsigned contract.
+- :guilabel:`Redirect Link`: Enter a redirect link for the employee to access the contract. A
+  redirect link takes the user from one URL to another, in this case, to the newly updated contract
+  specifically written for them.
+- :guilabel:`Who can Sign`: Select either *All Users* or *On Invitation*.
+
+   - :guilabel:`All Users`: Any user in the organization can sign the contract.
+   - :guilabel:`On Invitation`: Only users selected in this field can sign the contract.
+
+- :guilabel:`Invited Users`: Select the person(s) that can sign the document.
+- :guilabel:`Document`: The attached document can be replaced by clicking the
+  :guilabel:`🖉 (Select)` icon. A pop-up window appears so another document can be selected for
+  upload. The file must be a PDF. To remove the document, click the :guilabel:`🗑️ (Clear)` icon.
+
+Once the edits are complete, click the :guilabel:`Save` button. All the information for the selected
+contract template populates the fields in the :guilabel:`Salary Information` tab. Any additional
+tabs, such as :guilabel:`Personal Documents`, appears if applicable.
+
+Salary information
+------------------
+
+.. image:: contracts/salary-info.png
+   :align: center
+   :alt: Optional tabs for a new contract.
+
+This section is where the specific salary details are defined. This section is country-specific, so
+depending on where the company is located, these fields may vary.
+
+Enter the amount in the various fields, or check a box to apply a benefit. Some options that can be
+entered here include :guilabel:`Meal Vouchers`, :guilabel:`Fuel Card`, :guilabel:`Internet`,
+:guilabel:`Paid Time Off`, etc.
+
+Some fields may be automatically filled in based off of the contracts selected in the
+:guilabel:`Contract Details` tab.
+
+Attachment of salary
+--------------------
+
+.. image:: contracts/salary-attachment.png
+   :align: center
+   :alt: Optional tabs for a new contract.
+
+Any automatic deductions or allocations for an employee, such as child support payments and wage
+garnishments, are referred to as a *salary attachment*. This section is where all of these
+deductions or allocations are set.
+
+To add a new deduction, click :guilabel:`Add a line`. Type in a description for the allocation under
+:guilabel:`Description`.
+
+.. image:: contracts/garnishment.png
+   :align: center
+   :alt: Enter a new line for each type of garnishment.
+
+Select the :guilabel:`Garnished Type` from the drop-down menu. Chose from:
+
+- :guilabel:`Attachment of Salary` - Any payments taken out towards something that is *not* child
```

Comment:
The dashes in this bulleted list should all be colons to match the formatting in the rest of the doc.
```suggestion
- :guilabel:`Attachment of Salary`: Any payments taken out towards something that is *not* child
```